### ISSUE TITLE
WEB_ROOT is never referenced

### ISSUE THREAD

**Issue by yuppox** [lvntky/nexus.c#3]
According to `README.md`:

> `WEB_ROOT`: The directory where the server looks for static files (default: "./public").

This is not accurate. `WEB_ROOT` does not exist in the code at all and is absent from `config.h`.

**Comment by lvntky** [comment]
Yeah i think i wrote it as a todo but forgot to implement :D I can implement this or we can work together @yuppox
